tasks/ast_codegen/src/main.rs

```diff
@@ -217,9 +217,6 @@ pub struct CliOptions {
     /// Runs all generators but won't write anything down.
     #[bpaf(switch)]
     dry_run: bool,
-    /// Don't run cargo fmt at the end
-    #[bpaf(switch)]
-    no_fmt: bool,
```

Comment:
LGTM, But please keep this option. It wasn't added because of the issue at hand.